kahuna/public/js/components/gr-tooltip/gr-tooltip.js

```diff
@@ -20,6 +18,21 @@ tooltip.directive('grTooltip', ['onValChange', function (onValChange) {
                     element.attr('data-title', newTooltip);
                 }));
             }
+
+            element.bind('mouseenter', () => {
+                $scope.addTimer = $timeout(() => {
+                    element.addClass('titip-default').addClass(`titip-${position}`);
+
+                    $scope.removeTimer = $timeout(() => {
+                        element.removeClass('titip-default').removeClass(`titip-${position}`);
+                    }, 3500);
```

Comment:
If several `mouseenter` end up fired, you will only have a reference to the last timeouts. You won't be able to cancel any previous pending timeout.

This seems to be getting a little complicated -- is there a genuine need for hiding the tooltip after a delay? :)